website/docs/index.html.markdown

```diff
@@ -179,13 +179,20 @@ In addition to [generic `provider` arguments](https://www.terraform.io/docs/conf
 
 For more information on configuring and managing programmatic API Keys see the [MongoDB Atlas Documentation](https://docs.atlas.mongodb.com/tutorial/manage-programmatic-access/index.html).
 
-## HashiCorp Terraform Version Requirement
-| MongoDB Atlas Provider version  | Required Terraform version |
-| ------------- | ------------- |
-| `1.12.0` and above  | `1.0+`  |
-| below `1.12.0`  | `0.13+` |
-
-* While we support minimum of HashiCorp Terraform version 1.0+ (as we have migrated to [Plugin Protocol Version 6](https://developer.hashicorp.com/terraform/plugin/terraform-plugin-protocol#protocol-version-6)), for your safety we strongly recommend only consuming versions of Terraform that are currently receiving Security/Maintenance Updates. For latest Terraform versions see: https://endoflife.date/terraform 
+## [HashiCorp Terraform Version](https://www.terraform.io/downloads.html) Compatibility Matrix
+ 
+| HashiCorp Terraform Release | HashiCorp Terraform Release Date  | HashiCorp Terraform Full Support End Date  | MongoDB Atlas Support End Date |
+|:-------:|:------------:|:------------:|:------------:|
+| 1.7.x | 2024-01-17 | 2026-01-31 | 2026-01-31 |
+| 1.6.x | 2023-10-04 | 2025-10-31 | 2025-10-31 |
+| 1.5.x | 2023-06-12 | 2025-06-30 | 2025-06-30 |
+| 1.4.x | 2023-03-08 | 2025-03-31 | 2025-03-31 |
+| 1.3.x | 2022-09-21 | 2024-09-30 | 2024-09-30 |
+| 1.2.x | 2022-05-18 | 2024-05-31 | 2024-05-31 |
```

Comment:
would it make sense to add one more line to explicitly call out that <= 1.1.x is no longer supported?

Reply:
Added additional sentence below such that we no longer have mention specific version numbers as discussed